app/assets/javascripts/app/form-validation.js

```diff
@@ -11,6 +11,17 @@ document.addEventListener('DOMContentLoaded', () => {
 
   if (forms.length !== 0) {
     [].forEach.call(forms, function(form) {
+      if (!form.hasAttribute('data-allow-double-submit')) {
```

Comment:
Do we need to check for this attribute if we aren't using it anywhere? Seems like we could exclude this conditional until we find a form where we want to allow double submission, and then add it there.